src/main/java/uk/co/evoco/webdriver/configuration/WebDriverConfig.java

```diff
@@ -74,15 +74,13 @@ public void setBaseUrl(String baseUrl) throws MalformedURLException {
     }
 
     /**
-     *
```

Comment:
These need adding back in - javadocs convention is to have a blank line first which is where the description goes

Reply:
added the blank line